lib/release-notes/2.0.0.md

```diff
@@ -0,0 +1,70 @@
+# v2.0.0 Release notes
+
+## Description
+
+This release is centered around removing global state.
```

Comment:
Even an intro, and a TL;DR! I love detailed major release notes like this 😄 👍

Reply:
@trygve-lie and I agreed on this approach a couple days ago while you were sick. Have been doing it for every module since 👍